content/20.results.md

```diff
@@ -168,14 +172,16 @@ A Welch two-sample t-test did not reveal any statistically significant differenc
 
```

Comment:
Do you want to note somewhere within this section that the US is held out from training of the classifier?

Reply:
Sorry, which section are you referring to? The name origin prediction model?

Reply:
Yep!

Reply:
Oh - in this section re: the application of that model to the US affiliated speakers

Reply:
Ah okay. I thought so but your comment was tagged at a different line. Will do!

Reply:
Should we mention this in Methods re training the Wiki2019-LSTM? I think the training process is a bit more general than just applying to US affiliated honorees.